user-guide/Advanced_Functionality/DataMiner_Systems/Running_BPA_tests.md

```diff
@@ -14,7 +14,10 @@ The *BPA* page shows a list of all the BPA tests that are currently available in
 
 - Regardless of how the list is grouped, the *Execute all* button in the top-right corner can be used to execute all available tests on all applicable Agents and clusters.
 
-For a group item, the *Status* column displays the number of individual entries in the group with a colored LED indicating their status: green for a successful test, red if issues were detected, and gray if no results are available. For each individual item within a group, this same column can display the following status information: *No result available*, *Test running*, *OK*, *Warning*, *Issues detected* or *Execution failed*.
+For a group item, the *Status* column displays the number of individual entries in the group with a colored LED indicating their status: green for a successful test, red if issues were detected, and gray if no results are available. For each individual item within a group, this same column can display the following status information: *No result available*, *Test running*, *OK*, *Warning*, *Issues detected*, *Execution failed*, or *Not applicable* (available from DataMiner 10.3.0 [CU18]/10.4.0 [CU6]/10.4.9 onwards<!--RN 39929-->).
```

Comment:
"and gray if no results are available" -> "and gray if the test is not applicable"

Reply:
```suggestion
For a group item, the *Status* column displays the number of individual entries in the group with a colored LED indicating their status: green for a successful test, red if issues were detected, light gray if no results are available, and dark gray if the test is not applicable. If the execution of a test has failed, a warning icon will be displayed in the *Status* column instead of a colored LED. For each individual item within a group, this same column can display the following status information: *No result available*, *Test running*, *OK*, *Warning*, *Issues detected*, *Execution failed*, or *Not applicable* (available from DataMiner 10.3.0 [CU18]/10.4.0 [CU6]/10.4.9 onwards<!--RN 39929-->).
```

As there are two types of gray, I believe this may be more accurate. I also noticed the warning icon in the *Status* column was not documented yet, so I've added this information as well. What do you think, @HansSolos?

Reply:
OK